starter-code/index.js

```diff
@@ -1,17 +1,57 @@
 class SortedList {
-  constructor() {}
+  constructor(items, length) {
+    this.items = [];
+    this.length = this.items.length;
+  }
 
-  add(item) {}
+  add(item) {
+  this.items.push(item);
+  this.length = this.items.length;
+  this.items.sort(function (a, b) {
+    return a - b
+  })
 
-  get(pos) {}
+  }
 
-  max() {}
+  get(pos) {
+    if (pos > this.length ) {
+      throw new Error("OutOfBounds");
+    } else {
+      return this.items[pos];
+    } 
+  }
 
-  min() {}
+  max() {
+    if (this.length === 0) {
+      throw new Error("EmptySortedList");
+    } else {
+      return Math.max(...this.items)
+  }
+}
+
+  min() {
+    if (this.length === 0) {
+      throw new Error("EmptySortedList");
+    } else {
+      return Math.min(...this.items);
+    }
+  }
 
-  sum() {}
+  sum() {
+    if (this.length === 0) {
+      return 0;
+    } else {
+     return this.items.reduce((acc, val) => acc + val)
+    }
+  }
 
-  avg() {}
+  avg() {
+    if (this.length === 0) {
+      throw new Error("EmptySortedList");
+    } else {
+      return (this.items.reduce((acc, val) => acc + val)) / this.items.length;
```

Comment:
you could have used sum() here for reusal instead of writing the same code again.
Still nice job! 
Keep it up!